### DISCUSSION TITLE
How frequent can Airbyte Cloud perform replication?

### DISCUSSION THREAD

**Discussion by pai911** [airbytehq/airbyte#28599] [Connector Questions] [answered]
In Airbyte OSS, I can set CRON to perform sync every 10 sec (even though the overhead + actual sync time will prevent it from start that frequently)

On Airbyte cloud UI, I saw a message like this:

<img width="1011" alt="image" src="https://github.com/airbytehq/airbyte/assets/6234559/e1657dce-90cd-4bdb-b689-a775965a7a87">

So I can assume that the fastest sync frequency is every hour by default.

But can anyone tell me what's the fastest sync frequency I can use on Airbyte Cloud (after contacting them) ?

**Comment by marcosmarxm** [comment, marked as answer]
Hello, the minimum sync frequency is one hour. For values less than that, the Airbyte team will evaluate each case individually.
